tests/test_utils.py

```diff
@@ -202,5 +204,21 @@ def test_multipleInputs(self):
         self.assertEqual(expected, files)
 
 
+class GlobToRegexTestCase(unittest.TestCase):
+
+    def testGlobList(self):
+        """Test that a list of glob strings converts as expected to a regex and
+        returns in the expected list.
+
+        This assumes the conversion algorithm of globToRegex produces the same
+        results as in the right side of the assert, and verifies the
+        globToRegex function api.
+        """
+        testval = ["foo*", "bar"]
+        self.assertEqual(globToRegex(testval),
```

Comment:
I think a better test is to check that the result of globToRegex really does work properly. ie does it return something that matches `bar` and not `bars` or `abar` and matches `food` and not `flood`.